Jellyfin.Server.Implementations/Migrations/20240914121116_MigrateUserParentalRating.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Jellyfin.Server.Implementations.Migrations
+{
+    /// <inheritdoc />
+    public partial class MigrateUserParentalRating : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.RenameColumn(
+                name: "MaxParentalAgeRating",
+                table: "Users",
+                newName: "MaxParentalRatingScore");
+
+            migrationBuilder.AddColumn<int>(
+                name: "MaxParentalRatingSubScore",
+                table: "Users",
+                type: "INTEGER",
+                nullable: true);
+
+            migrationBuilder.AlterColumn<string>(
```

Comment:
hmmmmmmm?

Reply:
I didn't change anything, this is what EFCore generated

Reply:
This can happen if a previous PR changed the model but forgot to generate a migration for it. It would be better to put this in a separate migration (outside this PR) with a relevant migration name.

The latest version of EF Core also has a feature that can be added to the CI process to prevent this from happening in the future: https://learn.microsoft.com/en-us/ef/core/what-is-new/ef-core-8.0/whatsnew#checking-for-pending-model-changes